bats/fb-content-katello.bats

```diff
@@ -82,7 +77,7 @@ setup() {
 @test "create activation key" {
   hammer -u admin -p changeme activation-key create --organization="Default Organization" \
     --name="Test AK" --content-view="Test CV" --lifecycle-environment="Test" \
-    --unlimited-hosts | grep -q "Activation key created"
+    --unlimited-content-hosts | grep -q "Activation key created"
```

Comment:
We have a pattern of adding version conditionals for versions in foreman-bats, though we haven't used it recently. Sounds like that would be useful here as well.